protected/assets/icons/svg/notification.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why use an SVG icon? Where dit it come from?

Reply:
That should have been removed...